packages/pds/src/api/app/bsky/actor/updateProfile.ts

```diff
@@ -14,15 +14,7 @@ export default function (server: Server, ctx: AppContext) {
   server.app.bsky.actor.updateProfile({
     auth: ctx.accessVerifierCheckTakedown,
     handler: async ({ auth, input }) => {
-      const requester = auth.credentials.did
-
-      const did = input.body.did || requester
```

Comment:
Would not mind getting rid of these `did` inputs!  I don't think we have a use for them now, and I think we sort of want to redesign that bit of auth anyway next time we come across the same use-case.

Reply:
ah yeah that's not a bad idea.

would clean things up & they're the type of param we can add in later (or solve w the auth system)